backend/src/routes/api/medication.routes.js

```diff
@@ -0,0 +1,17 @@
+import Medication, { medicationValidator } from '../../models/Medication.js';
+import { StatusCodes } from 'http-status-codes';
+
+const MedicationRoutes = (router) => {
+  router.delete('me/meds/:id', deleteMedication);
+};
```

Comment:
It seems that we already have this class in the router - please merge those 2 files